attachments/models.py

```diff
@@ -124,6 +126,7 @@ class Property (models.Model):
     model = models.CharField(max_length=200, blank=True, help_text='The path to the lookup model for a ModelChoiceField.')
     content_type = models.ManyToManyField(ContentType, related_name='attachment_properties', blank=True)
     required = models.BooleanField(default=True)
+    allowed_file_types = models.CharField(max_length=200, help_text='Space separated file types that are allowed for upload.', blank=True)
```

Comment:
Let's make this a TextField and indicate it's whitespace-separated (not just spaces). One per line is often more readable.